lessons/04-html-tables/index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <meta name="viewport" content="width=device-width, inital-scale=1.0" charset="utf-8" />
+            <title>Stanley Cup Champions</title>
+    </head>
+
+    <body>
+        <h1>Stanley Cup Champions</h1>
+        <h2>Every team that's won the cup </h2>
+        
+        <img src="https://nhl.bamcontent.com/images/photos/287516894/1024x576/cut.jpg" alt="image of the Stanley Cup" />
+
+        <h3>Champions</h3>
+
+        <table border="1">
+            <thead>
+                <tr>
+                    <th>Year</th>
+                    <th>Team</th>
+                    <th>Coach</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                <tr>
+                    <th>2017</th>
+                    <th>Pittsburgh Penguins</th>
+                    <th>Mike Sullivan</th>
+                </tr>
```

Comment:
So close!

Inside the `<tr>`'s that are in the `<tbody>` we want to use `<td>` for *table data* instead of `<th>` which stands for *table heading*